torch/distributed/fsdp/wrap.py

```diff
@@ -318,10 +317,13 @@ class ParamExecOrderWrapPolicy:
             wrapping each transformer layer into one FSDP unit, and can be easily combined with checkpointing
             within each transformer layer.
 
-        tracing_config (Optional[TracingConfig]):
-            The configuration used to perform symbolic tracing at FSDP constructor to get the module and
-            parameter execution order. If set as None, then symbolic tracing is not enabled, and one forward
-            as well as backward iteration are needed to get the parameter execution order.
+        tracing_config:
```

Comment:
I think we can leave the type annotation in the docstring but omit it from the code. This way we at least still know what the type should be.

Reply:
sounds good!